README.md

```diff
@@ -288,8 +288,6 @@ I would like to thank Team Microverse for giving me the oppurtunity to work in a
 
 ## 📝 License <a name="license"></a>
```

Comment:
Great job @OmarMWarraich 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.
- [ ] [Optional] You may also remove the second author from your readme file since this is a solo project.